devtools/debug.py

```diff
@@ -1,5 +1,6 @@
 import os
 import sys
+from pathlib import Path
```

Comment:
see #49 & #50, for startup performance reasons we defer as many imports as possible.

Please can you move this import down into the function where it's called.